pipelines/tekton/resources.go

```diff
@@ -78,27 +78,33 @@ func generatePipeline(f fn.Function, labels map[string]string) *pplnv1beta1.Pipe
 	// Deploy step that uses an image produced by S2I builds needs explicit reference to the image
 	referenceImageFromPreviousTaskResults := false
 
+	var tasks []pplnv1beta1.PipelineTask
```

Comment:
We should also update `func getPipelineName()` (down below in this file. To create an unique Pipeline for git/direct method.

Reply:
Is it needed?

Reply:
Imagine creating running on cluster build via git, so it will create corresponding pipeline (with 3 tasks incl `git-clone`), then he decided to switch to direct code approach -> it would use the same pipeline even though it should use another one with 2 tasks only.

Reply:
The pipeline is reused? I thought it would be created on each build.